iceoryx2-bb/log/src/lib.rs

```diff
@@ -150,22 +165,34 @@ pub mod log;
 pub mod fail;
 pub mod logger;
 
-use iceoryx2_pal_concurrency_sync::iox_atomic::IoxAtomicU8;
+#[cfg(feature = "std")]
+pub use from_env::{set_log_level_from_env_or, set_log_level_from_env_or_default};
```

Comment:
Fine for now. But can you create a task or a sub-issue to use `getenv` from `stdlib` in our platform abstraction. See also `iceoryx_platform`.

Reply:
@elBoberido Could you point me to this function? I might be blind but I cannot find it with `rg`

Reply:
@orecham can you create an issue for this?

Reply:
https://github.com/eclipse-iceoryx/iceoryx2/issues/1153